### PR TITLE
Remove assertk

### PR DESCRIPTION
**Pull request by abrooksv**
## Motivation and Context
Standardized and removed some tech debt

## Related Issue(s)
#128

## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: core/tst/software/aws/toolkits/core/utils/ZipUtilsTest.kt [hunk 1]
```diff
@@ -62,9 +61,9 @@ class ZipUtilsTest {
         ZipFile(zipFile.toFile()).use { actualZip ->
             val actualEntry = actualZip.entries().toList().find { it.name == "file.txt" }
 
-            assert(actualEntry).isNotNull()
+            assertThat(actualEntry).isNotNull
```
Comment (rli) [inline core/tst/software/aws/toolkits/core/utils/ZipUtilsTest.kt]: isNotNull `()`?

File: jetbrains-core/tst/software/aws/toolkits/jetbrains/utils/NotificationUtilsTest.kt [hunk 2]
```diff
@@ -32,11 +29,9 @@ class NotificationUtilsTest {
 
         NullPointerException().notifyError("ooops", project = project)
 
-        assert(notification).isNotNull {
-            assert(it.actual.content) {
-                startsWith("java.lang.NullPointerException")
-                contains("NotificationUtilsTest.kt")
-            }
-        }
+        assertThat(notification).isNotNull
```
Comment (rli) [inline jetbrains-core/tst/software/aws/toolkits/jetbrains/utils/NotificationUtilsTest.kt]: same here

File: jetbrains-core/tst/software/aws/toolkits/jetbrains/core/stack/PageButtonsTest.kt [hunk 3]
```diff
@@ -2,8 +2,7 @@
 // SPDX-License-Identifier: Apache-2.0
 package software.aws.toolkits.jetbrains.core.stack
 
-import assertk.assert
-import org.junit.Assert
```
Comment (kiiadi) [inline jetbrains-core/tst/software/aws/toolkits/jetbrains/core/stack/PageButtonsTest.kt]: we should create a ktlint rule that detects junit `Assert` (which is an abomination against mankind).

File: core/tst/software/aws/toolkits/core/utils/ZipUtilsTest.kt (hunk 1 above)
Comment (abrooksv) [reply core/tst/software/aws/toolkits/core/utils/ZipUtilsTest.kt]: Kotlin sees it as a property not a method

Comment (rli) [reply core/tst/software/aws/toolkits/core/utils/ZipUtilsTest.kt]: weird but ok

File: jetbrains-core/tst/software/aws/toolkits/jetbrains/core/stack/PageButtonsTest.kt (hunk 3 above)
Comment (abrooksv) [reply jetbrains-core/tst/software/aws/toolkits/jetbrains/core/stack/PageButtonsTest.kt]: We use it in few places so we need to fix those first

File: core/tst/software/aws/toolkits/core/utils/ZipUtilsTest.kt (hunk 1 above)
Comment (abrooksv) [reply core/tst/software/aws/toolkits/core/utils/ZipUtilsTest.kt]: It is because it matches boolean Java Bean property syntax for a boolean

**Pull request merged**